app/src/components/integrations/integrationProviders/ldapIntegration/ldapFormFields/ldapFormFields.jsx

```diff
@@ -110,17 +126,20 @@ const urlValidator = bindMessageToValidator(validate.ldapUrl, 'requiredFieldHint
 
 @connect((state) => ({
   passwordEncoderType: integrationFormValueSelector(state, PASSWORD_ENCODER_TYPE_KEY),
+  nameType: integrationFormValueSelector(state, NAME_TYPE_KEY),
 }))
 @injectIntl
 export class LdapFormFields extends Component {
   static propTypes = {
     intl: PropTypes.object.isRequired,
     initialize: PropTypes.func.isRequired,
     change: PropTypes.func.isRequired,
+    stepNumber: PropTypes.number.isRequired,
```

Comment:
We agreed that in case of changes in Class components they should be updated to Functional components.